src/hash.c

```diff
@@ -276,9 +277,17 @@ int verify_bundle_hash(struct manifest *manifest, struct file *bundle)
 			      current->last_change, current->filename);
 
 		if (!verify_file(bundle, local)) {
-			printf("Warning: hash check failed for Manifest.%s for version %i\n",
-			       current->filename, manifest->version);
-			ret = 0;
+			if (!force) {
+				printf("Warning: hash check failed for Manifest.%s for version %i\n",
+					   current->filename, manifest->version);
+				ret = 0;
+			} else {
+				string_or_die(&log_cmd, "echo \"swupd security notice:"
+					" --force used to bypass hash verification\" | systemd-cat");
+				(void) system(log_cmd);
+				free(log_cmd);
+				ret = 0;
+			}
```

Comment:
Quick note: This will confict with #125, which fixes the problem of corrupt or missing bundle manifests. 

No biggie, though, I can rebase if this is merge before, or help you rebase if that patch makes it in earlier.